cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -2500,12 +2506,15 @@ impl MachInst for Inst {
     }
 
     fn gen_constant<F: FnMut(RegClass, Type) -> Writable<Reg>>(
-        to_reg: Writable<Reg>,
-        value: u64,
+        to_regs: ValueRegs<Writable<Reg>>,
+        value: u128,
         ty: Type,
         mut alloc_tmp: F,
     ) -> SmallVec<[Self; 4]> {
         let mut ret = SmallVec::new();
+        let to_reg = to_regs
+            .only_reg()
+            .expect("multi-reg values not supported on x64");
```

Comment:
I imagine this is implemented in a subsequent PR?

Reply:
Indeed!